### ISSUE TITLE
Plushie missions

### ISSUE THREAD

**Issue by a-random-lemurian** [endless-sky/endless-sky#7006]
### Problem Description

This isn't a problem. But we need plushie missions.

### Related Issue Links

No results for "plushies".

### Desired Solution

Plushie missions in vanilla.

My plugin, LEESt (pronounced "least"), features the Sayari Plushie Authority, an organization that makes, distributes, and sells plushies of Sayari. However, I'm not asking to merge the entire SPA itself into vanilla (please don't).

Perhaps there should be a mission where you are rewarded with a plushie.

Any human plushies (like Freya Winters) are probably going to be hard since we might need to find an actress whose face we can 3D scan for the plushie model, which leaves us with the Hai.

The plushie may be of Sayari herself or just any random Hai in general. If we choose the Hai, then someone needs to create a model of a squirrel plushie (which already exists IRL).

I would be willing to do the mission text, maybe just not the model itself.

TLDR: Let's make a mission with a Hai plushie as reward.

### Alternative Approaches

Create the plushie mission, but just give the player some text that says "You put the plushie in your ship's storage room", but doesn't actually appear in the outfitter Specials menu.

**Comment by Zitchas** [comment]
a) Every plushie I've seen has only the most vague resemblence to the person in question. I don't think we need to get 3D scans of actors and actresses for them.

b) I don't think "we need plushies" is really a very useful issue. Or even a request. The lack of plushies in the game isn't particularly noticeable or much of a detriment.

That being said, if you want to submit one or more plushie related missions or content somewhere... I don't see any problem with that. Could be neat, and add a bit of fun.

**Comment by roadrunner56** [comment]
I don't think anyone is particularly dying to see plushies in Endless Sky. Please only use the Github Issues function to report bugs or feature requests.
